changelogs/client_server/newsfragments/2121.clarification

```diff
@@ -0,0 +1 @@
+Fix typo "not" -> "no" in "Secrets" -> "Key Storage"
```

Comment:
I'm not sure if this is documented anywhere but people commonly use the phrase

> Fix various typos throughout the specification.

so that multiple typo fixes only generate a single changelog entry per section (as in https://github.com/matrix-org/matrix-spec/blob/main/content/changelog/v1.14.md?plain=1#L27C1-L27C307, for instance).